.evergreen/config.yml

```
@@ -93,29 +93,23 @@ functions:
           go version
           go env
 
-          LIBMONGOCRYPT_TAG="1.6.1"
+          # TODO (GODRIVER-2676) use libmongocrypt 1.7.0 once it is released.
```

```suggestion
          # TODO(GODRIVER-2676): use libmongocrypt 1.7.0 once it is released.
```
[nit] we usually use this formatting for `TODO`s.